plan/plan.go

```diff
@@ -151,13 +152,22 @@ func Eval(c *v2.Config) (*Plan, error) {
 	return p, nil
 }
 
-// Print prints a plan
-func Print(p *Plan) error {
-	out, err := yaml.Marshal(p)
-	if err != nil {
-		return errs.WrapInternal(err, "yaml: could not marshal")
+// Print prints a plan based on fogg file type
+func Print(p *Plan, fileName string) error {
```

Comment:
this Print is for human consumption (and not related to the config file format since it is already printing yaml). I think it would be ok to assume for now that this can print yaml files and preserve the old behavior